packages/spruce-skill/config/auth.ts

```diff
@@ -10,10 +10,11 @@ const gql = (options: {
 }): string => `
 {
 	${
-		options.userId
+		options.userId && options.organizationId
```

Comment:
With further thought I think we need to adjust this to include the organizationId if it's set but not require it (and still load the `User` like before).  I'm thinking about a skill that enhances a page outside an organization like my personal profile page.

Reply:
Let me know what you think about the change I just pushed. Basically added a separate query for when there is a user id but no org id and removed UserLocations and UserOrganizations from that query since they come back null without an org id.